### DISCUSSION TITLE
Rooms are case-sensitive

### DISCUSSION THREAD

**Discussion by Yarmiplay** [Syncplay/syncplay#589] [Ideas]
**Describe the bug**
Rooms are case-sensitive. Editing the list on the side works, but it's still an "issue" to run into when you think you logged into the same room but don't see each other. Changing the case "outside" of the list also acts weird sometimes. Why not reduce every room to just be lowercase? Or otherwise ignore case-sensitivity with rooms, make "Yarmiplay" and "yarmiplay" be the same room.

**To Reproduce**
1. Go to 2 clients
2. Log into a room with the same name on both, but change one of the letter to a different case (upper case or lower case)
3. Not be in the same room
4. That's the issue

**Expected behavior**
Be in the same room despite having different cases when you wrote down the room name

**Screenshots**
Here you go:
![image](https://user-images.githubusercontent.com/6286286/196776501-127f3233-b8ac-4fba-95e7-b60d4ce1255b.png)

**Version and platform:**
I'm using Windows 10 with 1.6.9, I remember this on earlier version and think it's probably the same with other platforms, kind of a "server" thing.

**Additional context**
Forcing all rooms to lowercase would be a good enough fix.
If you want rooms to be case-sensitive or otherwise don't think it's really a bug or an issue, feel free to close this immediately :)

**Comment by Et0h** [comment]
I'm happy to hear people's views on whether they think the server should move people to an established (active/permanent/persistent) room in a case sensitive or a case-insensitive manner.

Reasons in favour of the server remaining to be case sensitive:
* Any server-side changes would only apply to servers running the new server, meaning behaviour would be different depending on what server you were connecting to.
* It would take development time to create and test the changes.

Reasons in favour of the server being changed to be case insensitive:
* There is unlikely to be a use-case when people want two names with the same name but different case.
* It is likely that people would expect to be connected to the room with the same name (ignoring case).

https://github.com/Syncplay/syncplay/issues/403 was implemented to make the join combo boxes case sensitive to match the server behaviour, but the server being case sensitive is  more an artefact of how it was coded than a specific decision.
